modules/tinymce/src/core/test/ts/browser/util/QuirksFirefoxTest.ts

```diff
@@ -80,11 +81,21 @@ describe('browser.tinymce.core.util.QuirksFirefoxTest', () => {
 
   it('TINY-9694: dragend should fire when drop fires with an image', () => {
     const editor = hook.editor();
-    const transfer = new window.DataTransfer();
-    transfer.setData('text/html', '<img src="test">');
-    editor.dispatch('drop', new window.DragEvent('drop', {
-      dataTransfer: transfer
-    }));
-    assertEvents([ 'drop', 'dragend' ]);
+    editor.setContent(`
```

Comment:
I don't think it needs to be a table I think you could just have two paragraphs and drag/drop between them.